### PR TITLE
fix(CI/CD): Enable code coverage for unit tests

### PR DESCRIPTION
**Pull request by jarifibrahim**
Currently, we are running codecov on GKE (fabric8-cd). This PR enables codecov on CICO builds

### CODE REVIEW

**Comment by fabric8cd** [comment]
@jarifibrahim fabric8/fabric8-ui:SNAPSHOT-PR-2572-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2572-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@jarifibrahim fabric8/fabric8-ui:SNAPSHOT-PR-2572-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2572-fabric8-planner.badger.fabric8.io

File: cico_setup.sh [hunk 1]
```diff
@@ -46,6 +46,7 @@ function build_planner() {
 function run_unit_tests() {
     # Run unit tests
     docker exec $CID npm run tests -- --unit
+    docker exec $CID "bash <(curl -s https://codecov.io/bash) -t 73933b5a-4aba-4b55-8612-a809ca4ada30"
```
Comment (baijum) [inline cico_setup.sh]: The token is a secret, right?
What about using `CODECOV_TOKEN` environment variable to set token?

Comment (jarifibrahim) [reply cico_setup.sh]: This isn't a secret actually. It's the `repo token`. This token is used to distinguish one repository from another.

**Comment by rgarg1** [comment]
Thanks for getting this done @jarifibrahim

**Comment by fabric8cd** [comment]
@jarifibrahim fabric8/fabric8-ui:SNAPSHOT-PR-2572-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2572-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@jarifibrahim fabric8/fabric8-ui:SNAPSHOT-PR-2572-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2572-fabric8-planner.badger.fabric8.io

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @jarifibrahim for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by alien-ike** [comment]
Hey @baijum! It seems you tried to trigger `/ok-without-tests` command, but this will not have any effect due to insufficient permission. You have to be admin or requested reviewer or pull request approver, but not pull request creator for this command to take an effect.

**Pull request merged**